### ISSUE TITLE
bug in pytorch: sum over dimention is wrong (not only in colab)

### ISSUE THREAD

**Issue by zaco87** [googlecolab/colabtools#1864]
Bug report for Colab: http://colab.research.google.com/.

For questions about colab usage, please use [stackoverflow](https://stackoverflow.com/questions/tagged/google-colaboratory/).

- Describe the current behavior:
- Code
- The dim used in sum with pytorch is doing the opposite of what it should.
- this is an example code from :
- https://colab.research.google.com/drive/1CXSzT9vyAeuAjs1L5Kef1WbVj0yWjY57#scrollTo=LlmsYJWUE2r3

x = torch.tensor([[1, 2, 3], 
                             [4, 5, 6]], dtype=torch.float32)
print('Original tensor:')
print(x)

# We can sum over each row:
print('\nSum of each row:')
print(torch.sum(x, dim=0))
print(x.sum(dim=0))

# Sum over each column:
print('\nSum of each column:')
print(torch.sum(x, dim=1))
print(x.sum(dim=1))

- Results:
Sum of each row:
tensor([5., 7., 9.])
tensor([5., 7., 9.])

Sum of each column:
tensor([ 6., 15.])
tensor([ 6., 15.])

- Describe the expected behavior:
- It should have been the opposite...
- The matrix is 2x3, so if you want to sum over each row (dim=0), you should get only 2 results, not 3
- Same goes the other way: sum over columns (dim=1) should give 3 results (as there are 3 columns) but gives 2

- The web browser you are using (Chrome, Firefox, Safari, etc.):
Chrome
- Link (not screenshot!) to a *minimal, public, self-contained* notebook that
  reproduces this issue (click the *Share* button, then *Get Shareable Link*):

**Comment by craigcitro** [comment]
If this is a pytorch issue, you should file in on their issue tracker.
